pylops/waveeqprocessing/kirchhoff.py

```diff
@@ -363,6 +381,17 @@ def _traveltime_table(z, x, srcs, recs, vel, y=None, mode="eikonal"):
 
         return trav, trav_srcs, trav_recs, dist
 
+    def _wavelet_reshaping(self, wav, dt):
+        """Apply wavelet reshaping as from theory in [1]_"""
+        f = np.fft.rfftfreq(len(wav), dt)
+        W = np.fft.rfft(wav, n=len(wav))
+        if self.ndims == 2:
+            Wfilt = W * np.abs(2 * np.pi * f)
```

Comment:
abs is not needed here since f comes from rfftfreq (positive)

Reply:
Good point :) 

If you agree, I am tempted to merge this and then start a new PR with some extra improvements on the Kirchhoff operator. I started looking into the Safron, 2018 thesis you mentioned, but I don't think we should add more here...